packages/overlay/src/HoverController.ts

```diff
@@ -11,13 +11,16 @@ governing permissions and limitations under the License.
 */
 
 import { conditionAttributeWithId } from '@spectrum-web-components/base/src/condition-attribute-with-id.js';
+import { isWebKit } from '@spectrum-web-components/shared';
 import { randomID } from '@spectrum-web-components/shared/src/random-id.js';
-
+import { noop } from './AbstractOverlay.js';
 import {
     InteractionController,
     InteractionTypes,
+    lastInteractionType,
 } from './InteractionController.js';
-import { noop } from './AbstractOverlay.js';
+
+export const SAFARI_FOCUS_RING_CLASS = 'remove-focus-ring-safari-hack';
```

Comment:
I wonder where can we best define this const if we're going to use it in multiple places in the project. (currently it is also defined in [`MobileController`](https://github.com/adobe/spectrum-web-components/blob/3d93d530a7c9691753338f82a1486ccc8abac10f/packages/picker/src/MobileController.ts#L18))

Reply:
maybe define and export it from the `InteractionController`?

Reply:
I think that is a good idea, unfortunately there are 2 `InteractionController`s, one for the Overlay and one for the Picker, resulting in the same double declaration problem.

Reply:
I'd still say let's keep this constant in the `InteractionController` for now (separately for both picker nd overlay) cz that is the entry point for the interactions related stuff for both overlay and picker.